components/mediation-ui/org.wso2.carbon.proxyadmin.ui/src/main/resources/web/proxyservices/index.jsp

```diff
@@ -586,6 +587,13 @@
                     }
                 }
             }
+            var serviceGroupElm = getElement("serviceGroup");
+            if(serviceGroupElm != null) {
+                if(serviceGroupElm.value != null && serviceGroupRegex.test(serviceGroupElm.value)) {
```

Comment:
We can do this in single if statement
if(serviceGroupElm != null && serviceGroupElm.value != null && serviceGroupRegex.test(serviceGroupElm.value))